### PR TITLE
Travis build troubleshooting

### PR DESCRIPTION
**Pull request by bluzarraga**

### CODE REVIEW

**Comment by bluzarraga** [comment]
@bitscuit @cheewaio this pr is ready for review, no rush though

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ NAMESPACE=ibm-common-services
 # Use your own docker registry and image name for dev/test by overridding the IMG and REGISTRY environment variable.
 IMG ?= ibm-helm-api-operator
 REGISTRY ?= quay.io/opencloudio
-CSV_VERSION ?= 3.6.0
+CSV_VERSION ?= 3.5.0 #TODO change this to 3.6.0 before merging
```
Comment (cheewaio) [inline Makefile]: we should keep it 3.6.0

File: deploy/olm-catalog/ibm-helm-api-operator/3.5.0/operator.ibm.com_helmapis_crd.yaml [hunk 2]
```diff
@@ -2,6 +2,11 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
```
Comment (cheewaio) [inline deploy/olm-catalog/ibm-helm-api-operator/3.5.0/operator.ibm.com_helmapis_crd.yaml]: We have to go back to change 3.5.0 CRD?

File: deploy/olm-catalog/ibm-helm-api-operator/0.0.1/operator.ibm.com_helmapis_crd.yaml [hunk 3]
```diff
@@ -2,6 +2,11 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
```
Comment (cheewaio) [inline deploy/olm-catalog/ibm-helm-api-operator/0.0.1/operator.ibm.com_helmapis_crd.yaml]: We have to go back to change 0.0.1 CRD?

File: deploy/crds/operator.ibm.com_helmapis_crd.yaml [hunk 4]
```diff
@@ -40,13 +45,33 @@ spec:
...
+                    pullPolicy:
+                      type: string
                     pullPolicy:
                       type: string                      
                     repository:
                       type: string
                     tag:
                       type: string
-              type: object
+                resources:
```
Comment (cheewaio) [inline deploy/crds/operator.ibm.com_helmapis_crd.yaml:56]: the merge here does look right? Did u try installing the CRD?

File: helm-charts/helm-api/values.yaml [hunk 5]
```diff
@@ -68,7 +68,8 @@ helmapi:
     mongoCaName: mongodb-ca-cert
     mongoClientCertMountPath: /certs/mongodb-client
     mongoClientCertName: mongodb-client-cert
-nodeSelector: {}
+nodeSelector:
```
Comment (cheewaio) [inline helm-charts/helm-api/values.yaml]: we should remove nodeSelector all together ..there have been 2 build break because of this

**Review by cheewaio — CHANGES_REQUESTED** [review]

**Pull request merged**